aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -5,288 +5,253 @@ description: Learn how to host an ASP.NET Core app in a Windows Service.
-* `{USER ACCOUNT}` &ndash; The user account (SID).
-* `(OI)` &ndash; The Object Inherit flag propagates permissions to subordinate files.
-* `(CI)` &ndash; The Container Inherit flag propagates permissions to subordinate folders.
-* `{PERMISSION FLAGS}` &ndash; Sets the app's access permissions.
-  * Write (`W`)
-  * Read (`R`)
-  * Execute (`X`)
-  * Full (`F`)
-  * Modify (`M`)
-* `/t` &ndash; Apply recursively to existing subordinate folders and files.
+## Log on as a service rights
 
-For the sample app published to the *c:\\svc* folder and the `ServiceUser` account with write/read/execute permissions, use the following command an administrative PowerShell 6 command shell.
+To establish *Log on as a service* rights for a service user account:
 
-```powershell
-icacls "c:\svc" /grant "ServiceUser:(OI)(CI)WRX" /t
-```
+1. Open the Local Security Policy editor by running *secpool.msc*.
+1. Expand the **Local Policies** node and select **User Rights Assignment**.
+1. Open the **Log on as a service** policy.
+1. Select **Add User or Group**.
+1. Provide the object name (user account) using either of the following approaches:
+   1. Type the user account (`{DOMAIN OR COMPUTER NAME\USER}`) in the object name field and select **OK** to add the user to the policy.
+   1. Select **Advanced**. Select **Find Now**. Select the user account from the list. Select **OK**. Select **OK** again to add the user to the policy.
+1. Select **OK** or **Apply** to accept the changes.
 
-For more information, see [icacls](/windows-server/administration/windows-commands/icacls).
+## Create and manage the Windows Service
 
-## Create the service
+### Create a service
 
-Use the [RegisterService.ps1](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/host-and-deploy/windows-service/scripts) PowerShell script to register the service. From an administrative PowerShell 6 command shell, execute the script with the following command:
+Use PowerShell commands to register a service. From an administrative PowerShell 6 command shell, execute the following commands:
 
 ```powershell
-.\RegisterService.ps1 
-    -Name {NAME} 
-    -DisplayName "{DISPLAY NAME}" 
-    -Description "{DESCRIPTION}" 
-    -Exe "{PATH TO EXE}\{ASSEMBLY NAME}.exe" 
-    -User {DOMAIN\USER}
+$acl = Get-Acl "{EXE PATH}"
+$aclRuleArgs = {DOMAIN OR COMPUTER NAME\USER}, "Read,Write,ReadAndExecute", "ContainerInherit,ObjectInherit", "None", "Allow"
+$accessRule = New-Object System.Security.AccessControl.FileSystemAccessRule($aclRuleArgs)
+$acl.SetAccessRule($accessRule)
+$acl | Set-Acl "{EXE PATH}"
+
+New-Service -Name {NAME} -BinaryPathName {EXE FILE PATH} -Credential {DOMAIN OR COMPUTER NAME\USER} -Description "{DESCRIPTION}" -DisplayName "{DISPLAY NAME}" -StartupType Automatic
 ```
 
-In the following example for the sample app:
+* `{EXE PATH}` &ndash; Path to the app's folder on the host. Don't include the app's executable in the path. A trailing slash isn't required
```

Comment:
Example values for each of these?
`d:\myservice\`
`Contoso\ServiceUser`
`MyService`
`d:\myservice\myservice.exe`
`"My sample service"`
`"My Service"`